File: web/application/models/Grupo.php
<?php

class Grupo {
	private $id;
	private $titulo;
	private $tipo;
	private $data;
	private $id_campo = array();

	public function getId() {
	    return $this->id;
	}

	public function setId($id) {
	    return $this->id = $id;
	}

	public function getTitulo() {
	    return $this->titulo;
	}

	public function setTitulo($titulo) {
	    return $this->titulo = $titulo;
	}

	public function getTipo() {
	    return $this->tipo;
	}

	public function setTipo($tipo) {
	    return $this->tipo = $tipo;
	}

	public function getData() {
	    return $this->Data;
	}

	public function setData($Data) {
	    return $this->Data = $Data;
	}

	public function getIdCampo() {
	    return $this->id_campo;
	}

	public function setIdCampo($id_campo) {
	    return $this->id_campo = $id_campo;
	}
}

File: java/src/com/ifsp/integrado/model/Grupo.java
package com.ifsp.integrado.model;

import java.util.ArrayList;
import java.util.Date;

public class Grupo {
	private int id;
	private String titulo;
	private char tipo;
	private Date data;
	private ArrayList<Campo> id_campo;

	public int getId() {
		return id;
	}
	
	public void setId(int id) {
		this.id = id;
	}
	
	public String getTitulo() {
		return titulo;
	}
	
	public void setTitulo(String titulo) {
		this.titulo = titulo;
	}
	
	public char getTipo() {
		return tipo;
	}
	
	public void setTipo(char tipo) {
		this.tipo = tipo;
	}
	
	public Date getData() {
		return data;
	}
	
	public void setData(Date data) {
		this.data = data;
	}
	
	public void setIdCampo(Campo campo) {
		this.id_campo.add(campo);
	}
	
	public ArrayList<Campo> getIdCampo() {
		return id_campo;
	}
	
}
